src/ffi/credential.rs

```diff
@@ -125,7 +125,11 @@ pub extern "C" fn anoncreds_create_credential(
             cred_request.load()?.cast_ref()?,
             cred_values.into(),
             rev_reg_id,
-            rev_status_list.load()?.cast_ref().ok(),
+            rev_status_list
+                .opt_load()?
+                .as_ref()
+                .map(AnonCredsObject::cast_ref)
+                .transpose()?,
```

Comment:
```suggestion
            rev_status_list
                .opt_load()?
                .cast_ref()
                .transpose()?,
```

Would this not work?

Reply:
If I do it directly it gives me a compilation error:

```
error[E0599]: no method named `cast_ref` found for enum `std::option::Option` in the current scope
```

Reply:
Okay yeah that makes sense. I think your implementation is good.